### ISSUE TITLE
Video Timeouts on Protect 1.20

### ISSUE THREAD

**Issue by simplytoast1** [hjdhjd/homebridge-unifi-protect#590]
**Describe The Problem:**

Since the update to Protect 1.2.0 my cameras have been timing out with no specific error (UVC-G3-Instant Stopped Video Streaming Session.

**To Reproduce:**

Stream a video for 30 or so minutes

**Logs:**

```
Show the Homebridge logs here.
Remove any sensitive information.
```

**Homebridge Configuration:**

```json
Show the relevant portion of your homebridge config.json here, if needed.
Remove any sensitive information.
```

**Screenshots:**

**Environment:**

* **Homebridge Version:**  1.3.8
* **Node Version:**  16.13
* **Homebridge-UniFi-Protect Plugin Version:** 4.4.3
* **UniFi Protect Version:**  1.2.0
* **UniFi Protect Controller Type:**  UDMP
* **UniFi Protect Controller Firmware Version:**  4.46.13
* **Apple Device and iOS / macOS / iPadOS / tvOS Version:** iOS 15.1.1
* **Operating System and OS Version:**  macOS

**Comment by hjdhjd** [comment]
I haven’t seen an issue, but I also am not running a stream for 30+ minutes. I’d love to hear the context / use case if you don’t mind? 😄 

As to the issue…I suspect it’s more a problem on the Protect end than the plugin…nothing’s changed here, but I’ll take a look over the next couple of weeks as time allows.

**Comment by simplytoast1** [comment]
Absolutely!  I use the G3 as a baby monitor, so I've used it for extended viewing for the last year or so.  Seems this issue cropped up in the latest version of Protect so likely what happened is 1) There is an undocumented time out or 2) There was a change in the camera firmware that causes this timeout to occur.

The ONLY reason I went to submit a bug report is that using the navicular web viewer in protect or the protect app I don't get these timeouts.

I was looking also into my UNIFI logging and saw nothing more then a disconnect notice as well so SOMEWHERE the connection is being dropped with no error.

I'm digging a bit too and let you know if I uncover anything.

As a side note, the new G4 instant works great with your plugin as well!

Hope that helps and thanks as always!
